src/ExpenseTrackerGrupo2/Persistence/Database/Scripts/init.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TABLE IF NOT EXISTS Users (
+    Id SERIAL PRIMARY KEY,
+    Name VARCHAR(100) NOT NULL,
+    Email VARCHAR(100) NOT NULL UNIQUE,
+    PasswordHash VARCHAR(255) NOT NULL,
+    CreatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS Expenses (
+    Id SERIAL PRIMARY KEY,
+    UserId INT NOT NULL,
+    Amount DECIMAL(10, 2) NOT NULL,
+    Description VARCHAR(255),
+    Source VARCHAR(100),
+    Month INT NOT NULL,
```

Comment:
Don't forget to consider the types we have defined on the backend.

Reply:
i bassed on the document, pls review database design

Reply:
Are you sure about it? Expense table doesn't Month column.
![image](https://github.com/user-attachments/assets/8c54ba5f-acd1-4677-91de-4780d72695a5)